ajax/libs/react-extras/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "react-extras",
+  "title": "react-extras",
+  "description": "Useful components and utilities for working with React",
+  "author": {
+    "name": "Sindre Sorhus",
+    "url": "sindresorhus.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sindresorhus/react-extras"
+  },
+  "npmName": "react-extras",
+  "npmFileMap": [
+    {
+      "basePath": "source",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "index.min.js",
```

Comment:
File doesn't exist in `source` (dir doesn't exist), nor does this file exist in `dist`.